src/js/components/Button/stories/CustomThemed/Active.js

```diff
@@ -1,26 +1,318 @@
+  text: {
+    xsmall: {
+      size: '14px',
+      height: '16px',
+      maxWidth: '379px',
+    },
+    small: {
+      size: '16px',
+      height: '20px',
+      maxWidth: '379px',
+    },
+    medium: {
+      size: '18px',
+      height: '24px',
+      maxWidth: '432px',
+    },
+    large: {
+      size: '24px',
+      height: '32px',
+      maxWidth: '538px',
+    },
+    xlarge: {
+      size: '30px',
+      height: '36px',
+      maxWidth: '643px',
+    },
+    xxlarge: {
+      size: '36px',
+      height: '40px',
+      maxWidth: '854px',
+    },
+    '3xl': {
+      size: '42px',
+      height: '46px',
+    },
+    '4xl': {
+      size: '48px',
+      height: '48px',
+    },
+    '5xl': {
+      size: '72px',
+      height: '72px',
+    },
+    // flatten to match 5xl
+    '6xl': {
+      size: '72px',
+      height: '72px',
+    },
+  },
+  button: {
+    icon: {
+      size: {
+        small: '16px',
+        medium: '18px',
+        large: '24px',
+      },
+    },
+    default: {
+      color: 'text-strong',
+      border: {
+        radius: '100px',
+      },
+      font: {
+        weight: 700,
+      },
+    },
+    primary: {
+      border: {
+        radius: '100px',
+      },
+      color: 'text-primary-button',
+      font: { weight: 'bold' },
+      extend: (props) => primaryBackground(props),
+    },
+    secondary: {
+      border: {
+        color: 'brand',
+        width: '2px',
+        radius: '100px',
+      },
+      color: 'text-strong',
+      font: {
+        weight: 700,
+      },
+    },
+    'cta-primary': {
+      extend: (props) => primaryBackground(props),
+    },
+    hover: {
+      primary: {
+        extend: (props) => primaryHoverBackground(props),
+      },
+      'cta-primary': {
+        extend: (props) => primaryHoverBackground(props),
+      },
+    },
+    disabled: {
+      opacity: 0.3,
+      // overriding what is currently in grommet-theme-hpe
+      background: undefined,
+      color: undefined,
+      primary: undefined,
+      secondary: undefined,
+      toolbar: undefined,
+      'cta-primary': undefined,
+      'cta-alternate': undefined,
+    },
+    size: {
+      small: {
+        pad: {
+          vertical: '7px',
+          horizontal: '18px',
+        },
+        // should we allow this or handle it in pad like above?
+        // are there postives/negatives in alignment in cases where
+        // button label wraps?
         font: {
-          family: `-apple-system, BlinkMacSystemFont`,
-        },
-      },
-      button: { default: {} }, // enabling kind button functionality
-    }}
-  >
-    <Box pad="large" align="center">
-      <Button hoverIndicator="light-1" onClick={() => {}} active>
-        {/*  When kind Button include children, it is treated as plain */}
-        <Box pad="small" direction="row" align="center" gap="small">
-          <Add />
-          <Text>Kind</Text>
-        </Box>
-      </Button>
+          height: '24px',
```

Comment:
Noticing this comment and the height value. Is this working? I don't see the effect coming through. Also don't see it represented in base.d.ts or base.js.

Reply:
Thanks for catching, this story was staged with the brand refresh theme and not intended to stay when we merge. The line-height question can be handled in a separate PR if necessary. Reverting the story to the original.